include/fiction/utils/layout_utils.hpp

```diff
@@ -283,68 +287,178 @@ Lyt normalize_layout_coordinates(const Lyt& lyt) noexcept
 
     return lyt_new;
 }
-
 /**
- * Converts the coordinates of a given cell-level layout to SiQAD coordinates. A new equivalent layout based on SiQAD
- * coordinates is returned.
+ * This function converts an SiDB cell-level layout into a lattice layout according to the provided orientation. It
+ * transfers charge states and physical parameters from the input layout to the lattice counterpart and handles the
+ * migration of SiDB defects if they exist.
```

Comment:
Given this explanation, it does not seem like this function is doing a lot of computation, it is just copying data around, is that correct? If so, that seems to be the job of a copy-constructor/copy-assignment operator.